core/frontend/src/tile/GltfReader.ts

```diff
@@ -1190,7 +1190,11 @@ export abstract class GltfReader {
 
     }
 
-    return new DisplayParams(DisplayParams.Type.Mesh, color, color, 1, LinePixels.Solid, FillFlags.None, renderMaterial, undefined, hasBakedLighting, textureMapping);
+    let width = 1;
+    if (isPointPrimitive && undefined !== material.extensions?.BENTLEY_materials_point_style)
+      width = (material.extensions.BENTLEY_materials_point_style as any)?.diameter;
```

Comment:
```suggestion
    const pointStyle = material.extensions?.BENTLEY_materials_point_style;
    if (pointStyle && pointStyle.diameter > 0 && Math.floor(pointStyle.diameter) === pointStyle.diameter) {
      width = pointStyle.diameter;
    }
```

Cast to `any` unnecessary if you define the type in `GltfSchema`.
Enforce the non-negative integer constraint from the specification.